.gitignore

```diff
@@ -1,2 +1,9 @@
-# Dotenv configurations
-**/.env
\ No newline at end of file
+**/.env

```

Comment:
Don't mess with the main gitignore unless the file to be ignored should be in the root directory.